client/src/components/molecules/ImgDropAndCrop/ImgDropAndCrop.jsx

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable react/jsx-props-no-spreading */
```

Comment:
why?

Reply:
I don't know how to resolve. Would appreciate your input here.

Reply:
yeah you are using from the library you can skip it.